source/features/repo-age.tsx

```diff
@@ -43,11 +43,13 @@ const getRepoAge = async (commitSha: string, commitsCount: number): Promise<[str
 };
 
 const getFirstCommit = cache.function(async (): Promise<[string, string] | undefined> => {
-	const commitInfo = await elementReady<HTMLAnchorElement | HTMLScriptElement>('a.commit-tease-sha, include-fragment.commit-tease');
+	const commitInfo = await elementReady<HTMLAnchorElement | HTMLScriptElement>('.commit-tease a[href*="/commit/"]');
 	const commitUrl = commitInfo instanceof HTMLAnchorElement ? commitInfo.href : commitInfo!.src;
 	const commitSha = commitUrl.split('/').pop()!;
 
-	const commitsCount = looseParseInt(select('li.commits .num')!.textContent!);
+	// In "Repository refresh" layout, the number of commits may not be rendered yet
+	const commitsCountLink = select('li.commits .num') ?? await elementReady('.commit-tease + * a[href*="/commits/"] strong');
```

Comment:
The selectors don't select a link
```suggestion
	const commitsCountElement = select('li.commits .num') ?? await elementReady('.commit-tease + * a[href*="/commits/"] strong');
```